src/java.desktop/share/classes/javax/swing/plaf/basic/BasicFileChooserUI.java

```diff
@@ -404,20 +410,23 @@ protected void installStrings(JFileChooser fc) {
         updateButtonText = UIManager.getString("FileChooser.updateButtonText",l);
         helpButtonText   = UIManager.getString("FileChooser.helpButtonText",l);
         directoryOpenButtonText = UIManager.getString("FileChooser.directoryOpenButtonText",l);
+        customApproveButtonText = UIManager.getString("FileChooser.customApproveButtonText", l);
```

Comment:
Not sure if it needs to be addressed throughout the file, but for the "L" var at the end of each getString(), sometimes there's a space after the comma and sometimes there's none. Maybe it's better to follow the same convention throughout the file. Maybe it's better to just copy the lines above the one's you added to match the code block only.

Reply:
Yeah, updated.